src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -994,4 +1018,30 @@ private static CompoundNBT writeLeafSaplingEntryToNBT(final BlockState state, fi
     {
         return new Tuple<>(NBTUtil.readBlockState(compound), new ItemStorage(ItemStack.read(compound), false, true));
     }
+
+    /**
+     * Load free blocks and pos from the config and add to colony.
+     */
+    private void discoverFreeBlocksAndPos()
+    {
+        for (final String s : MinecoloniesAPIProxy.getInstance().getConfig().getCommon().freeToInteractBlocks.get())
```

Comment:
We are in the process of moving this kind of stuff out of configs and into tags.

Please adapt this old code to the new format.

Reply:
I thought about this, but we got blocks and positions here + this here is important for servers to be backwards compatible. I can add a todo for 1.17 though.

Reply:
I'm not sure if we really want to move these block permission exceptions stuff really over to tags